docs/cugraph/source/nx_cugraph/supported-algorithms.rst

```diff
@@ -0,0 +1,356 @@
+Supported Algorithms
+=====================
+
+The nx-cugraph backend to NetworkX connects
+`pylibcugraph <../../readme_pages/pylibcugraph.md>`_ (cuGraph's low-level Python
+interface to its CUDA-based graph analytics library) and
+`CuPy <https://cupy.dev/>`_ (a GPU-accelerated array library) to NetworkX's
+familiar and easy-to-use API.
+
+Below is the list of algorithms that are currently supported in nx-cugraph.
+
+
+Algorithms
+----------
+
+The list of currently supported algorithms is as follows:
```

Comment:
"list of ... currently supported" is written twice in quick succession here. I found it redundant.

Reply:
Removed 2nd instance